File: Flink1.12/src/main/scala/com/MuziMin/source_02/bean/Order.scala
package com.MuziMin.source_02.bean

/**
 * @author : 李煌民
 * @date : 2021-02-25 15:01
 *       ${description}
 **/
case class Order(
                  id: String,
                  userId: Int,
                  money: Int,
                  createTime: Long
                ) {
  override def toString: String = {
    s"""{"id":"$id","userId":"$userId","money":"$money","createTime","$createTime"}"""
  }
}

File: SpringStudy/src/main/java/com/muzimin/bean/Order.java
package com.muzimin.bean;

/**
 * @author: 李煌民
 * @date: 2022-06-14 23:21
 **/
public class Order {
    private String orderName;
    private int price;
    private String address;

    public Order(String orderName, int price) {
        this.orderName = orderName;
        this.price = price;
    }

    public Order(String orderName, int price, String address) {
        this.orderName = orderName;
        this.price = price;
        this.address = address;
    }

    public Order() {
    }

    public String getOrderName() {
        return orderName;
    }

    public void setOrderName(String orderName) {
        this.orderName = orderName;
    }

    public int getPrice() {
        return price;
    }

    public void setPrice(int price) {
        this.price = price;
    }

    public String getAddress() {
        return address;
    }

    public void setAddress(String address) {
        this.address = address;
    }

    @Override
    public String toString() {
        return "Order{" +
                "orderName='" + orderName + '\'' +
                ", price=" + price +
                ", address='" + address + '\'' +
                '}';
    }
}
